packages/homepage/src/components/LoadInView.js

```diff
@@ -1,40 +1,26 @@
-import { debounce } from 'lodash-es';
-import React, { useCallback, useEffect, useRef, useState } from 'react';
-
-import getScrollPos from '../utils/scroll';
-
-const LoadInView = ({ children, style, ...props }) => {
-  const [show, setShow] = useState(false);
-  const el = useRef(null);
-  const elPos = useRef(null);
-
-  useEffect(() => {
-    elPos.current = el.current.getBoundingClientRect().top;
-  }, []);
-
-  const listenFn = useCallback(() => {
-    if (!show && elPos.current && getScrollPos().y + 600 > elPos.current) {
-      requestAnimationFrame(() => {
-        setShow(true);
-      });
-    }
-  }, [show]);
-
-  useEffect(() => {
-    const listen = debounce(listenFn, 50);
-
-    window.addEventListener('scroll', listen);
-
-    return () => window.removeEventListener('scroll', listen);
-  }, [listenFn]);
+import React from 'react';
+import { useInView } from 'react-hook-inview';
+import { motion } from 'framer-motion';
 
+const LoadInView = ({ children }) => {
+  const [ref, inView] = useInView({ unobserveOnEnter: true });
   return (
-    <div
-      ref={el}
-      style={{ display: 'inline-block', width: '100%', ...style }}
-      {...props}
-    >
-      {show && children}
+    <div ref={ref}>
+      {inView && (
+        <motion.section
```

Comment:
This animation should be disabled if the user has enabled _reduce motion_, you can use `useMatchMedia`, for a11y

Reply:
This is an amazing idea!

Want to work with me on some checklist for accesability?

Reply:
Sure thing!

Reply:
Updated! Ill draft up a checklist!